index.bs

```diff
@@ -72,9 +72,9 @@ always put user needs above all.
 Similarly, when beginning to design an API,
 be sure to understand and document the user need that the API aims to address.
 
-[The internet is for end users](https://intarchboard.github.io/for-the-users/draft-iab-for-the-users.html):
```

Comment:
Why not leave this as a link (but now linking to the RFC), for ease of use, since we have the biblio link in the See also?

Reply:
Fixed in 01b9f86.